TACC/Frontera-Portal
# Task/FP-664 - Add system name to directory creation modal.

## Overview: ##
Pulled in system information to display the system name in the mkdir modal.

## PR Status: ##


## Related Jira tickets: ##

* [FP-664](https://jira.tacc.utexas.edu/browse/FP-664)

## Summary of Changes: ##

## Testing Steps: ##
1. Open the Add Folder modal in either of the data files system paths.
2. Check that the system's name appears before the path in the modal header

## UI Photos:
**BEFORE**
<img width="771" alt="mkdir_before" src="https://user-images.githubusercontent.com/29575979/101416529-abf00c00-38af-11eb-8717-5a2edb5104c5.png">
_The close button was removed in PR, #270; no such code change is in the diff of this PR, #273._ — @tacc-wbomar 

**AFTER**
![image](https://user-images.githubusercontent.com/29575979/102378039-fb5aca00-3f8a-11eb-9bc9-058849c444f7.png)
